src/utils/imageCompress.ts

```diff
@@ -0,0 +1,8 @@
+export enum AvailableForCompressExtension {
+    PNG = 'png',
+    JPG = 'jpg',
+    JPEG = 'jpeg',
+}
+
+export const canBeCompressed = (fileName: string) =>
```

Comment:
I'm not sure about this name, maybe it can be like `isCompressible`

Reply:
ok